homeassistant/components/life360/strings.json

```
@@ -14,7 +14,7 @@
       "invalid_username": "Invalid username",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "user_already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
```

This is the preferred key: 
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
```